# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/test/crashtests/small-timebase.mp4
# Commit: 0d9c1ac74d9c
# Full Hash: 0d9c1ac74d9cf147aa99fad91c7e1eca0e24fa1b
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-02 04:43:27
# Description:
#   Bug 1874723 - Accept imprecision when trimming audio content media in certain cases. r=alwu
#   
#   In the test case, the time base is 69 (generated by a fuzzer), and so there's a
#   very large loss in precision in the computation, and eventually the assumption
#   doesn't hold. It should however hold for the common case.
# ==============================================================================

diff -r f84c76f9460d -r 0d9c1ac74d9c dom/media/test/crashtests/small-timebase.mp4
Binary file dom/media/test/crashtests/small-timebase.mp4 has changed
